src/main/scala/com/microsoft/hyperspace/index/rules/RuleUtils.scala

```diff
@@ -303,6 +305,7 @@ object RuleUtils {
       plan: LogicalPlan,
       useBucketSpec: Boolean,
       useBucketUnionForAppended: Boolean): LogicalPlan = {
+    val ci = index.derivedDataset.asInstanceOf[CoveringIndex]
```

Comment:
Is `ci` standing for current index? Could you provide a better, more meaningful name?

Reply:
Done. It's short for coveringIndex.